Dockerfile

```diff
@@ -75,6 +78,9 @@ RUN python3 -m ensurepip \
                 ./dist/prio-*.whl \
                 ./processor
 
+ENV SPARK_HOME=/usr/local/lib/python3.6/site-packages/pyspark
+ENV PYSPARK_PYTHON=python3
+
```

Comment:
I recently learned about multi-stage docker builds, which might let you consolidate some of the logic that is common to both issues, see this discussion on an iodide issue: https://github.com/iodide-project/iodide/issues/2156

(not a request to change anything here btw, just a suggestion for a minor improvement)